config/urls.py

```diff
@@ -18,9 +18,9 @@
 
 urlpatterns = [
     # To the Admin Url
-    path('admin/', admin.site.urls),
+    path("admin/", admin.site.urls),
     # To the Core Url
```

Comment:
Change this to `Core URLS`. Although it doesn' matter much as it's pretty much obvious.